src/run.js

```diff
@@ -325,6 +325,8 @@ const processPage = ({
           elem.tagName === 'STYLE' &&
           (!elem.type || elem.type.toLowerCase() === 'text/css');
         const isStylesheetLink = elem =>
+          elem.rel !== 'preload' &&
```

Comment:
`elem.rel !== 'preload' && elem.rel !== 'prefetch' && elem.rel.toLowerCase() === 'stylesheet` can we collapse this into `elem.rel.toLowerCase() === 'stylesheet'`? @peterbe do you remember why it was done this way?

Reply:
Actually, I don't see it like that. This massive if statement says that it needs to be either `<link rel="Stylesheet" href="whatever">` or `<link href="specifically.css">`

So if you have `<link rel="whatever" href="whatever.css">` it *would* be let through. 

I don't know why we allow `<link href="specifically.css">` but I guess it makes sense. If you don't specify the `rel` attribute out of "laziness" then it should work. 

I think the really important change would be to make it...
```javascript
elem.rel.toLowerCase() === 'stylesheet' || (
  !elem.rel && elem.href.toLowerCase().endsWith('.css')
)
```

I.e. the only time we allow it in is if the `href` attribute ends with `.css` if and only if the `rel` attribute is not set.

Reply:
Same with Google Chrome and Safari and Safari iOS. 

Basically, I'm inclined to revise this whole if statement to just be:
```javascript
const isStylesheetLink = elem =>
  elem.tagName === 'LINK' &&
  elem.href &&
  elem.rel.toLowerCase() === 'stylesheet' &&
  !(
      elem.href.toLowerCase().startsWith('data:') ||
      elem.href.toLowerCase().startsWith('blob:') ||
      elem.media.toLowerCase() === 'print'
  );
```

Reply:
In fact, we can simplify it even more:

```javascript
const isStylesheetLink = elem =>
  elem.tagName === 'LINK' &&
  elem.href &&
  elem.rel.toLowerCase() === 'stylesheet' &&
  !elem.href.toLowerCase().startsWith('data:') &&
  !elem.href.toLowerCase().startsWith('blob:') &&
  !elem.media.toLowerCase() === 'print';
```

What do you think @stereobooster ?

Reply:
Yes, I thought of the same thing. I just wonder why we didn't do it initially. I guess we'll never know

Reply:
Hello,

I found the time to update my branch.

I used the function you suggested: 
```js
const isStylesheetLink = elem =>
  elem.tagName === 'LINK' &&
  elem.href &&
  elem.rel.toLowerCase() === 'stylesheet' &&
  !elem.href.toLowerCase().startsWith('data:') &&
  !elem.href.toLowerCase().startsWith('blob:') &&
  !elem.media.toLowerCase() === 'print';
```

modified like this : 
```diff
-  !elem.media.toLowerCase() === 'print';
+  elem.media.toLowerCase() !== 'print';
```

All tests are passing. \o/